src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -319,14 +318,13 @@ internal XmlFormatClassWriterDelegate XmlFormatWriterDelegate
             }
             set
             {
-#if NET_NATIVE
                 _xmlFormatWriterDelegate = value;
-#endif
             }
         }
```

Comment:
There's no functional difference here, except removed a #if at the cost of adding a member for NetCore.